pkg/tag/item_test.go

```diff
@@ -0,0 +1,117 @@
+package tag
+
+import (
+	"testing"
+
+	"github.com/reearth/reearth-backend/pkg/id"
+	"github.com/stretchr/testify/assert"
+)
+
+var _ Tag = &Item{}
+
+func TestItemBuilder_NewID(t *testing.T) {
+	b := NewItem().NewID()
+	assert.NotNil(t, b.i.id)
```

Comment:
```suggestion
	assert.NotEqual(t, id.TagID{}, b.i.id)
```

This assertion always passes because `b.i.id` is not a reference.